frontend/public/models/index.ts

```diff
@@ -1165,6 +1165,19 @@ export const InfrastructureModel: K8sKind = {
   crd: true,
 };
 
+export const NetworkOperatorConfigModel: K8sKind = {
+  label: 'Network',
+  labelPlural: 'Networks',
+  apiVersion: 'v1',
+  apiGroup: 'config.openshift.io',
+  plural: 'networks',
+  abbr: '',
```

Comment:
You'll want to define an `abbr` that is used in the badge to the left of links. Even if you don't have a details page for this resource, the badge can still show up (e.g. in the search page).

Reply:
I know, but we haven't created this resource type (just added it here because it was missing and we need to access to it).

It seems the SDN team has not defined any abbreviation for this data type:

```
$ oc  api-resources | grep -E '(^NAME|networks)'
NAME                                  SHORTNAMES       APIVERSION                                    NAMESPACED   KIND
networks                                               config.openshift.io/v1                        false        Network
networks                                               operator.openshift.io/v1                      false        Network
```